src/addTest.js

```diff
@@ -91,10 +92,11 @@ define(['ModernizrProto', 'Modernizr', 'hasOwnProp', 'setClasses'], function(Mod
    * @optionProp addTest
    * @access public
    * @function addTest
-   * @param {string|object} feature - The string name of the feature detect, or an
+   * @param {string|Object} feature - The string name of the feature detect, or an
```

Comment:
why?

Reply:
The [check-types rule](https://github.com/gajus/eslint-plugin-jsdoc/blob/master/.README/rules/check-types.md) from jsdoc requires this uppercase notation, it wasnt checked though before since there was a empty line between the docs and the code. when removing it, eslint warned me: 
  81:3  warning  Invalid JSDoc @param "feature" type "object"  jsdoc/check-types